src/rules/css/detectBadMozBinding.js

```diff
@@ -2,16 +2,17 @@ import * as messages from 'messages';
 import { isLocalCSSUri } from 'utils';
 
 
-export function detectBadMozBindingURL(rule) {
+export function detectBadMozBindingURL(cssContent, filename=null,
+                                       {startLine, startColumn}={}) {
   var messageList = [];
-  for (let declaration of rule.declarations) {
+  for (let declaration of cssContent.declarations) {
```

Comment:
As mentioned in the outdated diff: I'm :+1: for this. Thanks and sorry for the :bike: :house: